src/utils/polygon.h

```diff
@@ -991,9 +992,40 @@ class Polygons
         }
     }
 
-    void fixSelfIntersections()
+    void resize(const coord_t& numerator, const coord_t divisor)
```

Comment:
Since the `Polygons` class generally behaves like a vector, it would perhaps be better to rename this to `scale` or something like that. The `resize` function of a vector changes the length of the vector.

Reply:
I'll have to look into this, since I'm not the original author. I think @rburema wrote it, maybe he can comment on this.

Reply:
Agreed will change